### PR TITLE
Saas 13.2 fix website form fja

### PR DESCRIPTION
**Pull request by fja-odoo**
Some fixes for the website form recently merged.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_form/static/src/snippets/s_website_form/options.js [hunk 1]
```diff
@@ -1222,6 +1247,53 @@ options.registry.AddField = FieldEditor.extend({
...
+                div.style.width = formOptions && formOptions.querySelector('we-customizeblock-options [data-select-style] input').value + 'px',
+                div.classList = 's_website_form_label';
+                this.$target[0].insertBefore(div, button);
+                break;
+            }
+            default:
+                button.classList.add(`float-${value}`);
+                break;
+        }
+    },
```
Comment (qsm-odoo) [inline addons/website_form/static/src/snippets/s_website_form/options.js]: Did not see that in fact, don't use floating stuff, it works kinda by chance. If the row was not using flex, end a form with a floating button would make overflow over the next snippet. In fact it is probably the case in some browser, I cannot find the spec that says floating stuff are cleared when combined with flex.

The option should toggle the text-left/text-right-text-center on the parent col (in fact, I wonder why you do it for center and not the other?)

File: addons/website_form/static/src/snippets/s_website_form/options.js [hunk 2]
```diff
@@ -1222,6 +1247,53 @@ options.registry.AddField = FieldEditor.extend({
...
+            case 'center':
+                this.$target[0].classList.add('text-center');
+                break;
+            case 'align': {
+                const div = document.createElement('div');
+                const formOptions = this.el.closest('.o_we_customize_panel');
+                div.style.width = formOptions && formOptions.querySelector('we-customizeblock-options [data-select-style] input').value + 'px',
+                div.classList = 's_website_form_label';
+                this.$target[0].insertBefore(div, button);
+                break;
```
Comment (qsm-odoo) [inline addons/website_form/static/src/snippets/s_website_form/options.js]: I wonder why you are only dealing with this div when we choose align... it should be dealt with every time I think?

File: addons/website_form/static/src/snippets/s_website_form/options.js (hunk 1 above)
Comment (fja-odoo) [reply addons/website_form/static/src/snippets/s_website_form/options.js]: Never used `text-left` and `text-right`... Now I know !

File: addons/website_form/static/src/snippets/s_website_form/options.js (hunk 2 above)
Comment (fja-odoo) [reply addons/website_form/static/src/snippets/s_website_form/options.js]: Well I remove it every time and only add it when align is selected as it is useless in the other cases.

Comment (qsm-odoo) [reply addons/website_form/static/src/snippets/s_website_form/options.js]: Ok, I did not see you remove it every time, my bad then

File: addons/website_form/static/src/snippets/s_website_form/options.js [hunk 3]
```diff
@@ -1247,6 +1244,49 @@ options.registry.AddField = FieldEditor.extend({
...
+    selectButtonPosition: function (previewMode, value, params) {
+        this.$target[0].querySelectorAll('.s_website_form_label').forEach(el => el.remove());
+        this.$target[0].classList.remove('text-center', 'text-left', 'text-right');
+        const button = this.$target[0].querySelector('.s_website_form_send');
+        switch (value) {
+            case 'align': {
+                const div = document.createElement('div');
+                const formOptions = this.el.closest('.o_we_customize_panel');
+                div.style.width = formOptions && formOptions.querySelector('we-customizeblock-options [data-select-style] input').value + 'px',
+                div.classList = 's_website_form_label';
```
Comment (qsm-odoo) [inline addons/website_form/static/src/snippets/s_website_form/options.js]: className

**Pull request closed without merging**